MapboxMobileEvents/Categories/NSProcessInfo+SystemInfo.m

```diff
@@ -0,0 +1,65 @@
+#import "NSProcessInfo+SystemInfo.h"
+#include <sys/types.h>
+#include <sys/sysctl.h>
+#include <mach/machine.h>
+
+@implementation NSProcessInfo (SystemInfo)
+
++ (NSString*)stringForProcessorType:(NSInteger)type subtype:(NSInteger)subtype {
```

Comment:
Note:
If type/subtype are not found in this sequence of checks, value will return an empty string ""